x-pack/plugins/security_solution/public/common/hooks/use_update_browser_title.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect } from 'react';
+import { navTabs } from '../../app/home/home_navigations';
+import { useIsGroupedNavigationEnabled } from '../components/navigation/helpers';
+import type { NavTab } from '../components/navigation/types';
+import { getLinkInfo } from '../links';
+import { useRouteSpy } from '../utils/route/use_route_spy';
+
+export const useUpdateBrowserTitle = () => {
+  const isGroupedNavEnabled = useIsGroupedNavigationEnabled();
+  const [{ pageName }] = useRouteSpy();
+  const linkInfo = getLinkInfo(pageName);
+
+  useEffect(() => {
+    if (!isGroupedNavEnabled) {
+      document.title = `${getTitle(pageName, navTabs)} - Kibana`;
+    } else {
+      document.title = `${linkInfo?.title ?? ''} - Kibana`;
```

Comment:
nit: Should we put only Kibana if linkInfo.title is undefined

Reply:
Sorry. I don't understand the suggestion. This code sets the page title `PageName - Kibana`. Like here:
<img width="226" alt="Screenshot 2022-07-26 at 10 19 50" src="https://user-images.githubusercontent.com/1490444/180959055-97b2c2c9-2e37-4b7d-9654-4f0669defe94.png">

`linkInfo?.title` and `getTitle` are null if the page is misconfigured. In that case, the title would be `- Kibana`.

The logic did not change in this PR. I only moved the code to a hook.

Reply:
I mean, if linkInfo?.title and getTitle are null, what about we put the title as `Kibana` without ` - `